### ISSUE TITLE
No module named 'umap.parametric_umap'

### ISSUE THREAD

**Issue by rggs** [lmcinnes/umap#542]
Hi, when trying to use parametric UMAP I get the following error:

`ModuleNotFoundError: No module named 'umap.parametric_umap'`

I installed TensorFlow, and when I run pip install umap-learn[parametric_umap] I get the following:

`Requirement already satisfied: umap-learn[parametric_umap] in /home/users/r******/anaconda3/envs/tf-gpu/lib/python3.8/site-packages (0.4.6)`

Followed by some other output about the requirements already installed. Any ideas what might be wrong here?

**Comment by nihcompmed** [comment]
You won't find the parametric_umap.py file in that site-packages folder. I had to download the Github code and move it into that location, then I renamed the pycache and then reran the calling script. It loads

import umap
from umap.parametric_umap import ParametricUMAP, load_ParametricUMAP

perfectly fine but I had trouble with actually running the example Jupyter notebook (see an issue I opened a few days ago). Not sure it's ready for public consumption yet.

Hope this helps.
